src/bitmap/inherent.rs

```diff
@@ -578,6 +578,80 @@ impl RoaringBitmap {
 
         None
     }
+
+    /// Removes the specified number of elements from the top.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use roaring::RoaringBitmap;
+    ///
+    /// let mut rb = RoaringBitmap::from_iter([1, 5, 7, 9]);
+    /// rb.remove_first(2);
+    /// assert_eq!(rb, RoaringBitmap::from_iter([7, 9]));
+    /// rb = RoaringBitmap::from_iter([1, 3, 7, 9]);
+    /// rb.remove_first(2);
+    /// assert_eq!(rb, RoaringBitmap::from_iter([7, 9]));
+    pub fn remove_first(&mut self, mut n: usize) {
+        if n > self.len() as usize {
+            return;
+        }
```

Comment:
I'd like to remove this `if`, and do something like `position = position.unwrap_or(self.containers.len())`, to remove all the containers in case we try to remove more than we have, although at the end, if we still have items to remove, we need to check if there _is_ a first container, because we may have removed them all.

Reply:
Thank you. This way I don't have to calculate the length in advance anymore!
https://github.com/RoaringBitmap/roaring-rs/pull/253/commits/6b54277b03167342e1eb99d7386dbcade19f0bc4